APIManager.js

```diff
@@ -0,0 +1,87 @@
+//This is the class that will manage all your APIs
+
+class APIManager {
+    constructor() {
+        this.data = {
+            mainUser: {},
+            userFriends: [],
+            pokemon: {},
+            about: {},
+            kanyeQuote: {}
+         }
+    }
+    
+    getQuote() {
+        $.ajax({
+            method: 'get',
+            url: 'https://api.kanye.rest/',
+            success: (kanyeQuote) => {
```

Comment:
you use `success` but you should also use `error` it is important to start error handling